doc/source/getting_started/docker.rst

```diff
@@ -124,3 +129,123 @@ with the `-np` switch.  For example:
 For additional command line arguments please see the ansys
```

Comment:
```suggestion
For additional command line arguments, see the Ansys
```

Reply:
Also, I've seen lots of references to Ansys Help guides or topics in the PyMAPDL doc. However, anyone without an Ansys user account will be unable to get to this content. I know that the PyFluent developers thought it was undesirable to including references that required Ansys account login. Should we work to remove these? (I know the best answer would be to open up the Ansys Help, but I'm not sure that Legal will ever let that happen!)

Reply:
I'd keep these in. Most likely these users will have access. If anything, they can be placeholders for when things are eventually made public. I remain hopeful.